docs/components/connectors/out-of-the-box-connectors/kafka-inbound.md

```diff
@@ -0,0 +1,143 @@
+---
+id: kafka-inbound
+title: Kafka Consumer Connector
+sidebar_label: Kafka Consumer Connector
+description: "The Kafka Consumer Connector allows you to connect your BPMN service with Kafka. Learn how to create a Kafka Consumer Connector and make it executable."
+---
+
+The **Kafka Consumer Connector** allows you to connect your BPMN service with [Kafka](https://kafka.apache.org/).
```

Comment:
I would add a few details here -- for example, inbound/outbound, and what you could achieve by connecting your BPMN service with Kafka. It may seem obvious to the user, but it does provide a clear sense of purpose.